metadata.rb

```diff
@@ -1,9 +1,11 @@
 name             'xcode'
-maintainer       'Urbandecoder Labs'
-maintainer_email 'jdunn@aquezada.com'
+maintainer       'ROBLOX, Inc'
+maintainer_email 'antek@roblox.com'
 license          'Apache 2.0'
-description      'Installs Apple XCode and command-line tools'
+description      'Provides LWRP to install Apple XCode'
```

Comment:
The term "LWRP" is deprecated and in particular you wrote a library here so let's just call it a custom resource.